Microsoft.Toolkit.Uwp.UI/Converters/CollectionEmptinessToObjectConverter.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Markup;
+
+namespace Microsoft.Toolkit.Uwp.UI.Converters
+{
+    /// <summary>
+    /// This class converts a collection size into an other object.
+    /// Can be used to convert to bind a visibility, a color or an image to the size of the collection.
+    /// </summary>
+    public class CollectionEmptinessToObjectConverter : IValueConverter
+    {
+        /// <summary>
+        /// Gets or sets the value to be returned when the collection is neither null nor empty
+        /// </summary>
+        public object NotEmptyValue { get; set; }
+
+        /// <summary>
+        /// Gets or sets the value to be returned when the collection is either null or empty
+        /// </summary>
+        public object EmptyValue { get; set; }
+
+        /// <summary>
+        /// Convert a boolean value to an other object.
+        /// </summary>
+        /// <param name="value">The source data being passed to the target.</param>
+        /// <param name="targetType">The type of the target property, as a type reference.</param>
+        /// <param name="parameter">An optional parameter to be used to invert the converter logic.</param>
+        /// <param name="language">The language of the conversion.</param>
+        /// <returns>The value to be passed to the target dependency property.</returns>
+        public object Convert(object value, Type targetType, object parameter, string language)
+        {
+            bool isEmpty = true;
+            var collection = value as IEnumerable;
```

Comment:
On the pro side : I don't like to enumerate multiple times an IEnumerable nor IEnumerable<T>.
On the con side : string is also IEnumerable<T> which was the original design, changing to ICollection<T>would be breaking and would make the user lose some scenarios.